charts/argo-cd/templates/argocd-server/role.yaml

```diff
@@ -23,6 +23,9 @@ rules:
   resources:
   - applications
   - appprojects
+  {{- if .Values.server.extensions.enabled }}
```

Comment:
@mkilchhofer Here is the Role and RoleBinding. It was just simplified because permissions are the same.

Reply:
Perfect 👍 thanks for the confirmation. LGTM